lib/std/build.zig

```diff
@@ -2075,6 +2076,9 @@ pub const LibExeObjStep = struct {
         if (self.link_eh_frame_hdr) {
             try zig_args.append("--eh-frame-hdr");
         }
+        if(self.link_emit_relocs){
+            try zig_args.append("-emit-relocs");
```

Comment:
Long options require two `-`

Reply:
Fixed this and the other issue in the newest commit